<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlightNavigator</class>
 <widget class="QWidget" name="HighlightNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="lb_title">
       <property name="text">
        <string>No Results</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tb_gotoPrevious">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset resource="hobbitscoreicons.qrc">
         <normaloff>:/hobbits-core/images/icons/arrow-left.png</normaloff>:/hobbits-core/images/icons/arrow-left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>9</width>
         <height>9</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_selected">
       <property name="text">
        <string>No Results</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tb_gotoNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="hobbitscoreicons.qrc">
         <normaloff>:/hobbits-core/images/icons/arrow-right.png</normaloff>:/hobbits-core/images/icons/arrow-right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>9</width>
         <height>9</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tw_highlights">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="hobbitscoreicons.qrc"/>
 </resources>
 <connections/>
</ui>
